doc/main.html

```diff
@@ -1076,10 +1076,11 @@ <h4>Formulae</h4>
         <h4>Special Characters</h4>
         <p>Special characters and/or character sets from languages other than base latin should be represented as thier actual unicode character, not as their HTML entity. 
         </p>
-          <div class="example">
-            <pre>
+        <div class="example">
```

Comment:
I would remove the example.

Reply:
updated in https://github.com/SMPTE/html-pub/pull/127/commits/0acf37cc942e5d23c753ab64c00ebe4f7ddadb95